tests/test_torch_losses.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+import numpy as np
+import tensorflow as tf
+import torch
+
+from dlomix.losses import masked_spectral_distance, masked_pearson_correlation_distance
+from dlomix.losses import masked_spectral_distance_torch, masked_pearson_correlation_distance_torch
+
+logger = logging.getLogger(__name__)
+
+
+# ------------------ intensity - masked spectral distance ------------------
+
+def test_tf_torch_equivalence_masked_spectral_distance():
+
+    y_true = [[0.1, 0.2, 0.3]]
+    y_pred = [list(reversed(y_true[0]))]
+
+    sa_tf = masked_spectral_distance(
+        tf.convert_to_tensor(y_true), 
+        tf.convert_to_tensor(y_pred)
+    )
+    sa_torch = masked_spectral_distance_torch(
+        torch.tensor(y_true), 
+        torch.tensor(y_pred)
+    )
+
+    logger.info(f"Spectral Angle: for tf: {sa_tf.numpy()} vs for torch: {sa_torch.numpy()}")
+
+    assert (sa_tf.numpy() == sa_torch.numpy()).all() # alternatively try np.array_equiv(A,B)
+
+
+# ------------------ intensity - masked pearson correlation distance ------------------
+
+def test_tf_torch_equivalence_masked_pearson_correlation_distance():
+
+    y_true = [[0.1, 0.2, 0.3]]
+    y_pred = [list(reversed(y_true[0]))]
+
+    pc_tf = masked_pearson_correlation_distance(
+        tf.convert_to_tensor(y_true), 
+        tf.convert_to_tensor(y_pred)
+    )
+    pc_torch = masked_pearson_correlation_distance_torch(
+        torch.tensor(y_true), 
+        torch.tensor(y_pred)
+    )
+
+    logger.info(f"Masked Pearson Correlation Distance: for tf: {pc_tf.numpy()} vs for torch: {pc_torch.numpy()}")
+
+    assert (pc_tf.numpy() == pc_torch.numpy()).all() # alternatively try np.array_equiv(A,B)
```

Comment:
Do you think `np.allclose`would be a better option? https://numpy.org/doc/stable/reference/generated/numpy.allclose.html

The idea would be to accomodate any floating point different that happens because of approxiamtion, casting, etc.. of the different frameworks.